pkg/cmd/server/api/v1/types.go

```diff
@@ -73,6 +73,9 @@ type NodeConfig struct {
 	// IPTablesSyncPeriod is how often iptable rules are refreshed
 	IPTablesSyncPeriod string `json:"iptablesSyncPeriod"`
 
+	// DisableUnidling controls whether or not the hybrid unidling proxy will be set up
+	DisableUnidling bool `json:"enableUnidling"`
```

Comment:
We have positive flags that default if not set.

```
EnableUnidling *bool `json:"enableUnidling"`
```

And add a defaulter in v1 to default nil to true.  The internal type does not need to be a bool pointer, just a bool.